.github/workflows/on-osx-latest.yml

```diff
@@ -0,0 +1,32 @@
+name: on-osx-latest
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+#      - name: Install GitHub CLI
+#        run: |
+#          brew update
+#          brew install gh
+      - name: compile oommf
+        run: |
+          cat oommf-version
+          pwd
+          ls -l
```

Comment:
This looks a lot like additional debug information. Do we still need `pwd` and `ls -l`? (The output is already quite long and there is probably no no benefit from either command.)

Reply:
Good point.